cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -1027,6 +1030,23 @@ void shouldUpdateNpiSubjectHeadingWhenMissing()
         assertThat(publication.getEntityDescription().getNpiSubjectHeading(), is(equalTo(expectedNpiSubjectHeading)));
     }
 
+    @Test
+    void shouldAddImportDetailWhenImportingCristinObject() throws IOException {
+        var cristinObject = CristinDataGenerator.randomObject();
+        var eventBody = createEventBody(cristinObject);
+        var sqsEvent = createSqsEvent(eventBody);
+        var publication = handler.handleRequest(sqsEvent, CONTEXT).getFirst();
+
+        var importDetail = publication.getImportDetails()
+                               .stream()
+                               .filter(f -> f.importSource().getSource().equals(Source.CRISTIN))
+                               .findFirst()
+                               .orElse(null);
+
+        assertNotNull(importDetail);
+        assertNull(importDetail.importSource().getArchive());
```

Comment:
Should add test to verify that `importDate` is not null?

Reply:
Probably?